deeppavlov/models/morpho_tagger/cells.py

```diff
@@ -12,24 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import keras.activations as kact
-import keras.backend as kb
-import keras.initializers as kinit
-import keras.layers as kl
 import numpy as np
-from keras.engine.topology import InputSpec
+import tensorflow.keras.backend as K
```

Comment:
One letter variables are misleading, switch back to `kb`.

Reply:
https://keras.io/backend/#using-the-abstract-keras-backend-to-write-new-code
It's how Keras recommends it, so no. We can replace `K` with `tf.keras.backend` and even aspire to replace those with native `tf` calls, but I'm not sure that it's something that we want to do now.

Reply:
I find no reason to denote ``tf.keras.backend`` with `K`. `K` is a common name for integer parameter, e.g. in loops like for ``for k in range(K)``.

Reply:
And global one-letter variables are a very bad habit anyway.

Reply:
Overruled, as this is a widely accepted notation.